sdks/java/extensions/sql/datacatalog/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/datacatalog/DataCatalogBigQueryIT.java

```diff
@@ -65,6 +66,11 @@ public void testReadWrite() throws Exception {
             "bigquery.`table`.`%s`.`%s`.`%s`",
             bqTable.getProjectId(), bqTable.getDatasetId(), bqTable.getTableId());
 
+    readPipeline
```

Comment:
Let us keep tests for both dialects. It is fine to copy/paste duplicate or break into a separate method with two `@Test` calls to it.

Reply:
I would suggest a to merge this PR firs than keeping both tests. It will be a following work to have tests for both dialects.